src/bika/lims/content/bikasetup.py

```diff
@@ -575,6 +575,23 @@ def getCounterTypes(self, instance=None):
                 " when 'Sampling workflow' is active")
         ),
     ),
+    BooleanField(
+        "DateSampledRequired",
+        schemata="Sampling",
+        default=True,
+        widget=BooleanWidget(
+            label=_(
+                "label_bikasetup_date_sampled_required",
+                default="Date sampled required"
+            ),
+            description=_(
+                "description_bikasetup_date_sampled_required",
+                default="Select this to make DateSampled field required on "
+                        "sample creation. This functionality only takes "
+                        "effect when 'Sampling workflow' is not active"
+            ),
+        ),
+    ),
```

Comment:
Please add the counterpart for this setting in the new SENAITE setup

Reply:
Addressed with https://github.com/senaite/senaite.core/pull/2609/commits/556e8bb5e4b2c2d51720b2cb68adf03dbd3739e8